tests/CliConfiguratorTest.php

```diff
@@ -121,7 +121,7 @@ public function testValidHelpers(): void
 
         $dbHelper = $helperSet->get('db');
         assert($dbHelper instanceof ConnectionHelper);
-        $this->assertInstanceOf(ConnectionHelper::class, $dbHelper);
+        $this->assertInstanceOf(ConnectionHelper::class, $dbHelper); // @phpstan-ignore-line
```

Comment:
Why not use baseline or ignore in phpstan config?

Reply:
I don't like putting this in the baseline, since for me the baseline is something that I'll drop and re-create with every major release. That way I can ensure to deal with BC breaking improvements for every major. This issue, however, is something that will stay probably forever, simply because PhpStan doesn't understand it.

I agree that putting it in PhpStan config would be better, I'll do that.